app/assets/stylesheets/header.scss

```diff
@@ -41,6 +41,10 @@ body {
 }
 
 @media(max-width: 992px) {
+  #about-img {
+    margin: 20px auto;
+    width: -webkit-fill-available;
```

Comment:
Hi @asaayushisingh nice approach! though using 'width: -webkit-fill-available' will be experienced only for chrome users, leaving out firefox, IE, and other users. good use will be to set.
 ```
    min-width: 100%;
     min-width: -moz-available;          
     min-width: -webkit-fill-available;  
     min-width: fill-available;
```

Also, Try to check this on other browsers before you make changes.

Reply:
Also ```width: 100%``` should be a good fix.

Reply:
Hey! @on2onyekachi  Thanks a lot for pointing this out.

Reply:
Hey! @on2onyekachi I had made the required changes. Kindly review it, Thank You!